graphql/schema.graphql

```diff
@@ -817,11 +817,11 @@ type Mutation {
 	"""
 	Adds a bookmark to the user's list of bookmarks.
 	"""
-	addBookmark(documentId: UUID!): String!
+	addBookmark(documentId: UUID!): AnnotatedDoc
```

Comment:
Why is the return type optional? What does it mean if this returns null?

If returning null is something that can really happen and means something, that should be a part of the docstring. If returning null means you gave a bogus document ID, I feel like that should be an error and not returning null. In that case I think you should make this return `AnnotatedDoc!` / remove the `Option<>` in your rust code, and handle the branch where we can't find the document.

Reply:
There is the option to return null because the document loader used in this returns an Option<AnnotatedDoc>

Reply:
The `ok_or` function might help. That turns an Option into a Result. That could quickly turn `None` into an error for us I think